03/app.js

```diff
@@ -1,58 +1,75 @@
 const books = {
-    '978-83-7278-000-3': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Philosopher\'s Stone',
-            pl: 'Harry Potter i Kamień Filozoficzny',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
+  '978-83-7278-000-3': {
+    author: 'J.K. Rowling',
+    title: {
+      en: "Harry Potter and the Philosopher's Stone",
+      pl: 'Harry Potter i Kamień Filozoficzny',
     },
-    '83-7278-007-2': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Chamber of Secrets',
-            pl: 'Harry Potter i Komnata Tajemnic ',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
+    translator: {
+      en: null,
+      pl: 'Andrzej Polkowski',
     },
-}
+  },
+  '83-7278-007-2': {
+    author: 'J.K. Rowling',
+    title: {
+      en: 'Harry Potter and the Chamber of Secrets',
+      pl: 'Harry Potter i Komnata Tajemnic ',
+    },
+    translator: {
+      en: null,
+      pl: 'Andrzej Polkowski',
+    },
+  },
+};
+
+books.getAuthor = function (isbn) {
+  // w tym przypadku this === books
+
+  if (typeof this[isbn] === 'undefined') {
+    // taki ISBN nie istnieje w moim spisie
+    return null;
+  }
 
-books.getAuthor = function(isbn) {
-    // w tym przypadku this === books
-    
-    if(typeof this[isbn] === 'undefined') {
-        // taki ISBN nie istnieje w moim spisie
-        return null;
-    }
+  const author = this[isbn]['author'];
+  if (author) {
+    // zwracam informacje o autorze
+    return author;
+  }
 
-    const author = this[isbn]['author'];
-    if(author) {
-        // zwracam informacje o autorze
-        return author;
-    }
+  // brak informacji o autorze
+  return false;
+};
 
-    // brak informacji o autorze
-    return false;
-}
+books.getTitle = function (isbn, lang) {
+  if (typeof this[isbn]['title'][lang] === 'undefined') {
```

Comment:
Warto sprawdzić jeszcze poprzedni element tj. "isbin" jeśli ten element nie istnieje to kod rzuci błędem i przestanie działać reszta. Obecne rozwiązanie sprawdza tylko czy `lang` istnieje. Dlatego trzeba najpierw wykonać sprawdzenie:
```
if(typeof this[isbn] !== 'undefined') { ... }
```